docs/source/tutorials_source/platform/tutorial_label_studio_export.py

```diff
@@ -0,0 +1,74 @@
+"""
+
+.. _lightly-tutorial-export-labelstudio:
+
+Tutorial 10: Export to LabelStudio
+=============================================
+
+This tutorial shows how you can easily label all images of a tag in Lightly
+using the open-source data labeling tool `LabelStudio <labelstud.io>`_
+
+What you will learn
+--------------------
+
+* Export a tag in the LabelStudio-compatible task description format
```

Comment:
Export a tag from Lightly

Reply:
Maybe also link to https://labelstud.io/guide/tasks.html#Basic-Label-Studio-JSON-format